package.json

```diff
@@ -87,27 +87,27 @@
     "winston-daily-rotate-file": "^4.5.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.12.10",
-    "@babel/plugin-transform-runtime": "^7.12.10",
-    "@babel/preset-env": "^7.12.11",
-    "@babel/preset-react": "^7.12.10",
-    "@babel/register": "^7.12.10",
-    "@babel/runtime": "^7.12.5",
+    "@babel/cli": "^7.13.0",
+    "@babel/plugin-transform-runtime": "^7.13.9",
+    "@babel/preset-env": "^7.13.9",
+    "@babel/preset-react": "^7.12.13",
+    "@babel/register": "^7.13.8",
+    "@babel/runtime": "^7.13.9",
     "@jest-runner/electron": "^3.0.0",
-    "@testing-library/dom": "^7.29.4",
+    "@testing-library/dom": "^7.29.6",
     "@testing-library/jest-dom": "^5.11.9",
-    "@testing-library/react": "^11.2.3",
-    "@testing-library/user-event": "^12.7.0",
+    "@testing-library/react": "^11.2.5",
+    "@testing-library/user-event": "^12.8.1",
     "babel-eslint": "^10.1.0",
     "dotenv": "^8.2.0",
-    "electron": "^11.2.1",
-    "electron-builder": "^22.9.1",
+    "electron": "^11.3.0",
+    "electron-builder": "^22.10.5",
     "electron-devtools-installer": "^3.1.1",
-    "eslint": "^7.18.0",
+    "eslint": "^7.21.0",
     "eslint-config-airbnb": "^18.2.1",
     "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jest": "^24.1.3",
-    "eslint-plugin-jsdoc": "^31.4.0",
+    "eslint-plugin-jest": "^24.1.5",
+    "eslint-plugin-jsdoc": "^31.6.1",
```

Comment:
That's a great question, haha. It could have been a few things. I believe I did an `npm update` followed by `npm install` when pulling down the latest `main`. I also installed and removed a few packages that I didn't end up using but checked out. So that also could have updated dependencies?

Okay, I just did an `npm update` on another branch and it looks like it made these exact version bumps. So that must have been it.